java/compaction/compaction-core/src/main/java/sleeper/compaction/strategy/impl/BasicLeafStrategy.java

```diff
@@ -64,11 +61,16 @@ public List<CompactionJob> createJobsForLeafPartition(Partition partition, List<
             if (filesForJob.size() >= compactionFilesBatchSize) {
                 // Create job for these files
                 LOGGER.info("Creating a job to compact {} files in partition {} in table {}",
```

Comment:
Logging out every new compaction job looks like it might be quite expensive sometimes?

Reply:
I've created the following issue to tidy up logging: https://github.com/gchq/sleeper/issues/2948